pages/docs/administration/getting-started/configuration/making-a-schema.mdx

```diff
@@ -0,0 +1,19 @@
+---
+priority: 200
+---
+
+import DocsWrapper from '@/src/docs/DocsWrapper'
+
+# Making a Schema
+
+export default ({ children }) => <DocsWrapper>{children}</DocsWrapper>
+
+- The schemas use a JSON schema (existing examples are in server/scripts/example_schemas)
```

Comment:
This needs to be expanded on much more. 

We should be detail why each property is used (like we are with widget) and also give other examples. Examples of how to make multiple choice, nested pages, dependencies, etc. 

We don't need to go into HUGE amounts of detail as we can forward users to the library's documentation, but as a user I'd expect a quick guide on how to make a very small schema from scratch.